catalog/ui/catalog-ui-search/src/main/webapp/js/ColorGenerator.js

```diff
@@ -30,12 +30,20 @@ define([
                 '#6a3d9a',
                 '#fdbf6f',
                 '#ff7f00',
-                '#fb9a99',
-                '#e31a1c',
                 '#b2df8a',
                 '#33a02c',
                 '#a6cee3',
-                '#1f78b4'
+                '#1f78b4',
```

Comment:
❓ I could be wrong, but I think we were trying to ensure these colors were colorblind safe.  Did you check for this when adding the new colors?

Reply:
The colors that were added came directly from @lcrosenbu , so I'm assuming so... But that's a great question. I'll find out.